modules/packages/ArgumentParser.chpl

```diff
@@ -155,12 +155,28 @@
     $ quickStart myFile --debug --optional
     optional expects 1 value
 
+  If a user asks for help with either ``-h`` or ``--help``, the following output
+  would be automatically generated and displayed:
+
+  .. code-block:: shell
+
+    $ quickStart -h
+    USAGE: quickStart <POSITIONAL> [-h, --help] [--debug] [--optional <OPTIONAL>]
+    
+    ARGUMENTS:
+            POSITIONAL
+            
+    OPTIONS:
+            -h, --help      Display this message and exit
+            --optional <OPTIONAL>
+
```

Comment:
Maybe it could say something about how the help message for each optional argument can be customized?

Reply:
Yep, I'll add a section for that!